tests/test_summary.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+import sys
+import pytest
+import pandas as pd
+from mock import patch
+sys.path.append('../')
+sys.path.append('../webviz_subsurface/containers/')
```

Comment:
Why are these two neeeded, compared to regular import?

Reply:
Removed